.github/workflows/package-build.yml

```diff
@@ -12,9 +12,9 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 jobs:
   package-build:
-    uses: tektronix/python-package-ci-cd/.github/workflows/_reusable-package-build.yml@v1.3.0
+    uses: tektronix/python-package-ci-cd/.github/workflows/_reusable-package-build.yml@v1.4.0
     with:
-      package-name: TekHSI
+      package-name: tekhsi
```

Comment:
This change should rename package name on pypi right.
![image](https://github.com/user-attachments/assets/325b9c06-2c40-4ef1-8e4a-c3c3b30e8bb8)

Reply:
No, this is the python importable package name, PyPI's names are case insensitive.